### PR TITLE
Home filter

### PR DESCRIPTION
**Pull request by kallycao**
Implemented filter dropdown,  default dog card view, and expanded dog card view.

### CODE REVIEW

File: frontend/components/App.jsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default function App() {
   //     "altitudeAccuracy":-1
   //   },
   //   "timestamp":1669174971110.204
-//   }
+  //   }
 
   let text = 'Waiting...';
   if (errorMsg) {
```
Comment (boyiq) [inline frontend/components/App.jsx:46]: Maybe this was from the previous PR logging whether the user granted location permission. Looks like it's only logging when there's a geolocation but not when there's an error message. Move console.log outside the else if.

**Review by boyiq — COMMENTED** [review]
The home page looks great!!! Everything looks good to me except one tiny thing

**Review by KateFeaster — APPROVED** [review]
Looks fantastic! Great work!

**Review by boyiq — APPROVED** [review]

File: frontend/components/home/MoreInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <Card width="100%" alignSelf="center" borderRadius="10">
+      <Text h4>{`${item.city}, ${item.state}`}</Text>
+      <View
+        style={{
+          flexDirection: 'row',
+          flexWrap: 'wrap',
+          justifyContent: 'left',
+        }}
+      >
+        <Button
```
[the remark this answers is not in the document]
Comment (kallycao) [reply frontend/components/home/MoreInfo.jsx:18]: Thank you Kate! I will be switching the buttons out for chips (https://reactnativeelements.com/docs/3.4.2/chip) when I begin my design phase! Buttons were the fastest way I could replicate the wireframe design.

**Pull request merged**